### PR TITLE
GEODE-1754: Cleaned up loadSharedConfigFromDir argument from InternalLocator.startLocator and its associated code calls.

### PR DESCRIPTION
**Pull request by ameybarve15**

### CODE REVIEW

File: geode-core/src/main/java/org/apache/geode/distributed/Locator.java [hunk 1]
```diff
@@ -253,7 +253,7 @@ private static Locator startLocator(int port, File logFile, InetAddress bindAddr
       java.util.Properties dsProperties, boolean peerLocator, boolean serverLocator,
```
Comment (bschuchardt) [inline geode-core/src/main/java/org/apache/geode/distributed/Locator.java]: Amey, there is also a loadSharedConfig static variable in Locator that should be removed.

File: geode-core/src/main/java/org/apache/geode/internal/DistributionLocator.java [hunk 2]
```diff
@@ -166,9 +166,9 @@ public void run() {
 
       try {
```
Comment (bschuchardt) [inline geode-core/src/main/java/org/apache/geode/internal/DistributionLocator.java]: There is also a static LOAD_SHARED_CONFIGURATION variable in this class that should be removed.

**Comment by bschuchardt** [comment]
Amey, if you don't mind I'm going to merge your commit with my fixes for GEODE-1893.  I'm removing the peerLocator & serverLocator options for starting a locator and it's hitting the same code you've modified.

**Comment by ameybarve15** [comment]
These changes have been merged in commit id: c33ce7db8148fe0b300066581f26582e3e044e96

**Comment by ameybarve15** [comment]
These changes have been merged in commit id: c33ce7db8148fe0b300066581f26582e3e044e96

**Pull request closed without merging**